docs/source/user-guide/binding/execution-engine.rst

```diff
@@ -79,6 +79,13 @@ The ExecutionEngine class
             or a object file instance. Object file instance is not usable after this
             call.
 
+   * .. method:: add_archive(archive_file)
+
+        Add the symbols from the specified static archive file to the execution
+        engine. It is a fatal error in LLVM if the *archive_file* does not exist.
+
+        * *archive* str: a path to the static object file
```

Comment:
The parameter name doesn't match that from above.  Is "static object file" appropriate here, or is "archive file" more consistent?

```suggestion
        * *archive_file* str: path to the archive file
```